src/Core/Repositories/SqlServer/CipherRepository.cs

```diff
@@ -88,6 +88,18 @@ public async Task<ICollection<CipherDetails>> GetManyByUserIdAsync(Guid userId,
             }
         }
 
+        public async Task<IEnumerable<CipherDetails>> GetManyDetailsByOrganizationIdAsync(Guid organizationId)
+        {
+            using (var connection = new SqlConnection(ConnectionString))
+            {
+                var results = await connection.QueryAsync<CipherDetails>(
+                        $"[{Schema}].[Cipher_ReadByOrganizationId]",
+                        new { OrganizationId = organizationId },
+                        commandType: CommandType.StoredProcedure);
+                return results;
+            }
+        }
+
```

Comment:
We already have a method for this that returns `IEnumerable<Cipher>`. Are you adding this endpoint just to cast it to `CipherDetails`? `CipherDetails` expects many more properties, so you cannot cast a sproc that returns a `Cipher` up to `CipherDetails` like this.

I recommend using the existing `IEnumerable<Cipher>` repo and just cast the user list down to `Cipher` in your services.

Reply:
Done